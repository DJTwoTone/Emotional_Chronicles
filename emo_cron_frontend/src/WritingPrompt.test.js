import React from 'react';
import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import WritingPrompt from './WritingPrompt';

it('should render', function() {
    render(
        <MemoryRouter>
            <WritingPrompt />
        </MemoryRouter>
    );
})

it('should match the snapshot', function() {
    const { asFragment } = render(
        <MemoryRouter>
            <WritingPrompt />
        </MemoryRouter>
    );
    expect(asFragment()).toMatchSnapshot();
})