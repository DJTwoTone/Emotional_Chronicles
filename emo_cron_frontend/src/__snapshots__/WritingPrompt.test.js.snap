// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot 1`] = `
<DocumentFragment>
  <div
    class="m-4 shadow card"
  >
    <div
      class="card-body"
    >
      <p
        class="display-5 lead card-text"
      >
        Something to write about...
      </p>
      <p
        class="display-4 card-text"
      >
        test prompt
      </p>
      <button
        class="btn btn-dark"
        type="button"
      >
        GET A NEW PROMPT
      </button>
    </div>
  </div>
</DocumentFragment>
`;
